.github/workflows/shared-publish-to-docker-versioned.yaml

```diff
@@ -42,6 +42,11 @@ on:
         description: If true, will attempt to publish any vulnerabilities to GitHub. Defaults to true. Set to false for private repos.
         type: string
         default: 'true'
+      slack_webhook:
+        description: 'The Slack webhook URL for sending messages via incoming webhooks, configured for #dev-uid2-alarms channel'
+        required: true
```

Comment:
If we set this to `required:true` it will break other pipelines using this workflow. Maybe instead provide it with a default value.